skiffWindowsApp/Skiff Desktop/MessageProcessor.cs

```diff
@@ -48,6 +48,9 @@ internal void CoreWebView2_WebMessageReceived(object sender, CoreWebView2WebMess
                         var notificationsPayload = JsonSerializer.Deserialize<NotificationDataWrapper>(receivedMessage.Data.ToString());
                         foreach (var notification in notificationsPayload.NotificationData)
                         {
+                            if (_consumedThreads.Contains(notification.ThreadId))
```

Comment:
I think this should be `messageID` based? you may want notifications for the same `threadID`

Reply:
IIRC all our desktop notification actions operate on threadIDs. Based on that, I not even implemented mailID as was not going to have any use, and introduce risk of mistakes. But if we do have it, I can do it.

Reply:
Oh, but I see your point on this one. I may want to look into the mailId to hide further notifications rather than the thread. Let me work on it a bit more.